app/DynamicsAdapter/DynamicsAdapter.Web.Test/Mapping/MappingProfile_SearchRequest_Test.cs

```diff
@@ -49,7 +49,7 @@ public void normal_SearchRequestOrdered_should_map_to_SearchRequestEntity_correc
                         RequestDate = new DateTimeOffset(2018, 9, 28, 0, 0, 0, new TimeSpan(1, 0, 0)),
                         Email = "agencyemail@test.com",
                         InformationRequested = new List<InformationRequested> { InformationRequested.Location, InformationRequested.PHN, InformationRequested.DL },
-                        LocationAddress = "NORTHERN AND INTERIOR CLIENT OFFICE, KAMLOOPS, BC"
+                        LocationCode = "NORTHERN AND INTERIOR CLIENT OFFICE, KAMLOOPS, BC"
```

Comment:
Did we not introduce code for location s?

Reply:
before, fmep send the whole location address, later, Dragan changed it to code (v, k etc.). So, at first, we have locationAddress, later, we add LocationCode.